sale_order_line_description/models/__init__.py

```diff
@@ -1,2 +1,4 @@
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
+
 from . import res_config_settings
-from . import sale_order_line
+from . import sale
```

Comment:
Why changing this as the file name should reflect the model name?

Reply:
@rousseldenis I change ```sale_order_line.py``` to ```sale.py``` because class ```SaleOrderLine``` is in ```sale.py``` in odoo13

Reply:
That's Odoo business :smile: In OCA, we use convention that says one file per model named as the model name. But that's nittpicking.

Reply:
@rousseldenis so, Should I change it's to original?

Reply:
That's better yes. Thanks

Reply:
Fixed it.